src/runtime/entries/netlify.ts

```diff
@@ -0,0 +1,59 @@
+import '#internal/nitro/virtual/polyfill'
+import type { Handler, HandlerResponse, HandlerContext, HandlerEvent } from '@netlify/functions/dist/main'
+import type { APIGatewayProxyEventHeaders } from 'aws-lambda'
+import { withQuery } from 'ufo'
+import { createRouter as createMatcher } from 'radix3'
+import { nitroApp } from '../app'
+import { useRuntimeConfig } from '../config'
+
+export const handler: Handler = async function handler (event, context) {
+  const config = useRuntimeConfig()
+  const routerOptions = createMatcher({ routes: config.nitro.routes })
+
+  const query = { ...event.queryStringParameters, ...event.multiValueQueryStringParameters }
+  const url = withQuery(event.path, query)
+  const routeOptions = routerOptions.lookup(url) || {}
```

Comment:
Note: Same issue mentioned in https://github.com/unjs/nitro/pull/538#discussion_r993783351 applies here.